akka-docs/src/main/paradox/stream/operators/Sink/actorRefWithAck.md

```diff
@@ -9,6 +9,29 @@ Send the elements from the stream to an `ActorRef` which must then acknowledge r
 Send the elements from the stream to an `ActorRef` which must then acknowledge reception after completing a message,
 to provide back pressure onto the sink.
 
+Messages sent to the actor materialized from this source will be emitted downstream, when there is demand. Otherwise, they will be buffered, up to the specified maximum, in conjunction with the overflow strategy.
```

Comment:
This documents `Source.actorRef`, right?

Reply:
thanks